CHANGELOG.md

```diff
@@ -8,6 +8,11 @@
   - Remove bitcore-message dependency.
   - Move feature to 'ecdsa-koblitz-signature-2016' package.
 
+### Changed
+  - **BREAKING**: `verify()`'s results.error is now always an error, not an
+    array of errors. In case of multiple errors, a wrapping error is thrown,
```

Comment:
My understanding is that a wrapping error is always thrown (which +1 to that consistency, btw). So this text should say something more like: "results.error is now always a `VerificationError` instance where `error.errors` is an array that includes all of the errors that occurred during the verification process."

Reply:
Thanks, changed!